frontend/views/components/Avatar.vue

```diff
@@ -1,11 +1,13 @@
 <template lang='pug'>
   img.c-avatar(
+    v-if='imageURL'
```

Comment:
Why is needed? Seemed to be working fine before?

Reply:
note the `v-else`. an image must always have a `src`, otherwise it's invalid. so, when it doesn't have a src, it only renders a placeholder (a div)